src/PowerTools/DbContextPackage.cs

```diff
@@ -520,7 +520,7 @@ private static void SetDataDirectory(Project project)
                 "DataDirectory",
                 project.IsWebProject()
                     ? Path.Combine(project.GetProjectDir(), "App_Data")
-                    : project.GetTargetDir());
+                    : Path.GetFullPath(project.GetTargetDir()));
```

Comment:
Should it be around the whole ternary expression? (Or even in `GetTargetDir` before it returns)